ngrinder-controller/build.gradle

```diff
@@ -46,12 +46,8 @@ dependencies {
     compile (group: "com.google.guava", name: "guava", version: "20.0")
     compile (group: "org.springframework.security", name: "spring-security-taglibs", version: spring_security_version)
 
-    compile (group: "org.apache.maven", name: "maven-embedder", version: "3.6.3")
-    compile (group: "org.apache.maven", name: "maven-compat", version: "3.6.3")
     compile (group: "org.eclipse.aether", name: "aether-connector-basic", version: "1.1.0")
     compile (group: "org.eclipse.aether", name: "aether-transport-wagon", version: "1.1.0")
```

Comment:
These two dependencies are also for the `maven-embedder`.

Reply:
Removed.